facades.md

```diff
@@ -0,0 +1,58 @@
+# Facade data in third-party-web
+
+In addition to identifying entities and products, third-party-web also includes data on available alternative libraries for products known as _facades_. A _facade_ is a static element which looks similar to the actual embedded resource, but is not functional and therefore much less taxing on the page load.
```

Comment:
Defining "facade" here seems redundant since we have the terminology section below.

Reply:
agree. @patrickhulce cool if we drop it here?  or we just inline the terminology into sentences up here.

Reply:
inline up here SGTM. I wanted to intro to the purpose of this document for the rest of the community and contributors that wasn't "Lighthouse has a new audit" :) 

terminology section was helpful for broad context google doc, but can just be in intro here SGTM